model-engine/model_engine_server/infra/gateways/resources/k8s_endpoint_resource_delegate.py

```diff
@@ -1146,12 +1496,43 @@ def _get_deployment_resource_name(request: CreateOrUpdateResourcesRequest) -> st
         deployment_resource_name = f"deployment-{flavor_class}-{mode}-{device}"
         return deployment_resource_name
 
+    @staticmethod
+    def _get_lws_resource_name(request: CreateOrUpdateResourcesRequest) -> str:
+        build_endpoint_request = request.build_endpoint_request
+        model_endpoint_record = build_endpoint_request.model_endpoint_record
+        flavor = model_endpoint_record.current_model_bundle.flavor
+        if isinstance(flavor, TritonEnhancedRunnableImageFlavor):
+            flavor_class = "triton-enhanced-runnable-image"
+        else:
+            flavor_class = "runnable-image"
+        if flavor_class == "triton-enhanced-runnable-image":
+            raise ValueError("LWS is not supported for Triton Enhanced Runnable Image")
+        # TODO we should really check this error earlier
```

Comment:
add the check on bundle creation? I guess regardless you'll have to handle the union type check here

Reply:
oh lmao oops the check actually does happen on endpoint create (e.g. `validate_bundle_multinode_compatibility()`)